sql-plugin/src/main/scala/com/nvidia/spark/rapids/ColumnarOutputWriter.scala

```diff
@@ -102,7 +102,7 @@ abstract class ColumnarOutputWriter(path: String, context: TaskAttemptContext,
       }
```

Comment:
2022 copyrights.

Reply:
Every file that was touched in 2022 got a 2022 copyright update.
```
$ git log sql-plugin/src/main/scala/com/nvidia/spark/rapids/ColumnarOutputWriter.scala
commit 3869bde288bbf178b56c2310b22d9b4b61009743
Author: Robert (Bobby) Evans <bobby@apache.org>
Date:   Fri Dec 31 09:14:06 2021 -0600

    Fix a metric and clean up timeline a little
```

I didn't think there was a requirement to go back and update the copyright for older files.

Reply:
But this wasn't even updated for 2021 copyrights when it was touched last year, and since we have to touch it this year to fix that, I just went with 2022.  If you want to make it 2021 I'm fine with that too.

Reply:
That is a great point. I missed that we missed the 2021 update. And now I have touched the file in 2022 so...